presto-docs/src/main/sphinx/release/release-318.rst

```diff
@@ -0,0 +1,73 @@
+===========
+Release 317
+===========
+
+General Changes
+---------------
+
+* Fix query failure when using ``DISTINCT FROM`` with the ``UUID`` type. (:issue:`1180`)
+* Improve performance for queries over ``information_schema`` that have a limit. (:issue:`999`)
+* Improve query performance when ``optimize_hash_generation`` is enabled. (:issue:`1071`)
+* Improve performance of information schema tables. (:issue:`1306`)
+* Add support for ``Asia/Qostanay`` time zone. (:issue:`1221`)
+* Add session properties for ``query_max_memory_per_node`` and
+  ``query_max_total_memory_per_node``. (:issue:`1212`)
+* Rename Kerberos properties to make them consistent with other
+  HTTP server properties. (:issue:`1270`)
+* Add :doc:`/connector/googlesheets`. (:issue:`1030`)
+* Change query CPU tracking for resource groups to update periodically while
+  the query is running. Previously, CPU usage would only update at query
+  completion. This improves resource management fairness when using
+  CPU-limited resource groups. (:issue:`1128`)
+* Add :func:`last_day_of_month` function. (:issue:`1295`)
+* Remove ``distributed_planning_time_ms`` column from ``system.runtime.queries``. (:issue:`1084`)
+* Change the semantics of the ``analysis_time_ms`` column in the ``system.runtime.queries``
+  table. Previously, it was the time spent on query planning. Now, it is the time spent on
+  semantic analysis of the query. (:issue:`1084`)
+* Add ``planning_time_ms `` column to the ``system.runtime.queries`` table that shows
+  the time spent on query planning. This is the same value that used to be in the
+  ``analysis_time_ms`` column. (:issue:`1084`)
+* Allow to display details of query that is in the queue or during semantic analysis.
+  Query details are accessible from the web ui or ``system.runtime.queries``. (:issue:`1079`)
+* Allow to cancel query that is in the queue or during semantic analysis. Query can
+  be cancelled from the web ui or using builtin procedure ``system.runtime.kill_query``.(:issue:`1079`)
+
+Hive Connector Changes
+----------------------
+
+* Fix query failure due to missing credentials while writing empty bucket files. (:issue:`1298`)
+* Fix bucketing of ``NaN`` values of ``real`` type. Previously ``NaN`` values
+  could be assigned a wrong bucket. (:issue:`1336`)
+* Fix reading collection delimiter set by Hive version earlier than 3.0. (:issue:`1321`)
+* Return proper error when selecting ``"$bucket"`` column from a table using
+  Hive bucketing v2. (:issue:`1336`)
+* Improve performance of S3 object listing. (:issue:`1232`)
+* Improve performance when reading data from GCS. (:issue:`1200`)
+* Add support for reading data from S3 Requester Pays buckets. This can be enabled
+  using the ``hive.s3.requester-pays.enabled`` configuration property. (:issue:`1241`)
+* Allow inserting into bucketed, unpartitioned tables. (:issue:`1127`)
```

Comment:
```suggestion
* Allow inserting into bucketed tables. (:issue:`1127`) (:issue:`1347`)
```

https://github.com/prestosql/presto/pull/1347

Reply:
This still applies, except that this should be reworded differently:

```suggestion
* Allow inserting into bucketed, unpartitioned tables. (:issue:`1127`)
* Allow inserting into existing partitions of bucketed, partitioned tables. (:issue:`1347`)
```

Reply:
This comment seems not to have been addressed.